crates/wasmparser/Cargo.toml

```diff
@@ -69,3 +70,8 @@ validate = [
 # Enable Serialize/Deserialize implementations for types in
 # `wasmparser::collections`
 serde = ['dep:serde', 'indexmap/serde', 'hashbrown/serde']
+
+# A feature that enables legacy exception handling proposal (phase 1)
+# support this feature as long as all leading browsers also support it
+# https://github.com/WebAssembly/exception-handling/blob/main/proposals/exception-handling/legacy/Exceptions.md
+legacy-exception-handling = []
```

Comment:
Ah I apologize I fear I may have misled you. Currently wasmparser doesn't have other Cargo features for wasm proposals so there's not a lot of infrastructure to support this and as you're discovered there's bugs and things to shake out. Given all that it's ok to remove this and have the support always be compiled-in-by-default.

What I moreso meant is that we'll want a new feature gate in `WasmFeatures` which is turned off at runtime by default but support for it is compiled in by default.

Reply:
Updated ✅